src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -7,8 +7,6 @@
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <UseECMAKey>true</UseECMAKey>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp'">$(DefineConstants);FEATURE_ZLIB</DefineConstants>
```

Comment:
It was, Crc32Helper.cs but that is why I deleted the code on that was no longer applicable on that file. Changes to that file are on commit 44d9f7f